pkg/ghclients/ghclients.go

```diff
@@ -80,13 +83,24 @@ func (g *GHClients) Get(i int64) (*github.Client, error) {
 	}
 	ctr := &httpcache.Transport{
 		Transport:           tr,
-		Cache:               httpcache.NewMemoryCache(),
+		Cache:               g.cache,
 		MarkCachedResponses: true,
 	}
 	g.clients[i] = github.NewClient(&http.Client{Transport: ctr})
 	return g.clients[i], nil
 }
 
+func (g *GHClients) LogCacheSize() {
```

Comment:
Nit: define a `getCacheSize()` fn on `memoryCache` struct and simply call that here.

Reply:
Will do. Then Items can be private.

Reply:
Done, PTAL.

Reply:
LGTM